dataRetriever/resolvers/epochproviders/nilEpochProvider.go

```diff
@@ -0,0 +1,33 @@
+package epochproviders
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data"
+)
+
+// NilEpochProvider will be used for regular validator/observer nodes
+type NilEpochProvider struct {
```

Comment:
disabled? and maybe unexported and use a constructor

Reply:
done, done and done